pkgs/development/python-modules/pylint/default.nix

```diff
@@ -74,6 +75,9 @@ buildPythonPackage rec {
   disabledTests = lib.optionals stdenv.isDarwin [
     "test_parallel_execution"
     "test_py3k_jobs_option"
+  ] ++ lib.optionals (pythonAtLeast "3.10") [
+    # https://github.com/PyCQA/pylint/issues/6296
+    "test_functional"
```

Comment:
Why don't we add `typing-extensions` to `checkInputs`?

Reply:
Yep, I have that already fixed up locally, will be part of the next push.